lib/computer.rb

```diff
@@ -0,0 +1,67 @@
+require './lib/player'
```

Comment:
I think what we have here is pretty good. This class functions, the only downside is it places an "x" instead of an "o".  I believe we intend to fix this in the next driver/navigator session.